packages/keystone/bin/commands/start.js

```diff
@@ -0,0 +1,28 @@
+const keystone = require('@keystone-alpha/core');
+const { executeDefaultServer, getEntryFileFullPath } = require('../utils');
+
+module.exports = {
+  // prettier-ignore
+  spec: {
+    '--port':   Number,
+    '-p':       '--port',
+    '--entry':  String,
+    '--out':    String,
+    '-o':       '--out',
+  },
+  help: ({ exeName }) => `
+    Usage
+      $ ${exeName} start --port=3000
+
+    Options
+      --port, -p  Port to start on [${keystone.DEFAULT_PORT}]
+      --entry     Entry file exporting keystone instance [${keystone.DEFAULT_ENTRY}]
+      --out, -o   Directory where build was saved [dist]
```

Comment:
Looks like `--out` isn't being used. It'll need to be picked up within `executeDefaultServer()`, and passed into `keystone.prepare()` inside `packages/keystone/bin/utils.js`.